frontend/public/components/start-guide.tsx

```diff
@@ -100,13 +100,14 @@ export const withStartGuide: WithStartGuide = (WrappedComponent, disable = true)
     if (flags[FLAGS.SHOW_OPENSHIFT_START_GUIDE]) {
       return (
         <>
-          <div className="co-m-pane__body">
-            <HintBlock title="Getting Started">
+          <Hint>
+            <HintTitle>{t('Getting Started')}</HintTitle>
```

Comment:
Nit:  we typically specify the namespace even though it's not required.
```suggestion
            <HintTitle>{t('public~Getting Started')}</HintTitle>
```